README.md

```diff
@@ -3,12 +3,48 @@
 The :computer: Nextcloud Desktop Client is a tool to synchronize files from Nextcloud Server
 with your computer.
 
+<p align="center">
```

Comment:
I forgot to mention this on the commit message. @jancborchardt suggested to add this image.

Reply:
:)